modules/lnd.nix

```
@@ -288,7 +324,7 @@ in {
     users.users.${cfg.user} = {
       isSystemUser = true;
       group = cfg.group;
-      extraGroups = [ "bitcoinrpc-public" ];
+      extraGroups = optional (cfg.backend == "bitcoind") "bitcoinrpc-public";
```

`optional` is very cryptic and it is not obvious it is adding an element into the list, suggesting using `optionals` instead:

```suggestion
      extraGroups = optionals (cfg.backend == "bitcoind") [ "bitcoinrpc-public" ];
```